frontend/components/equipment-card.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <v-card border="md" @click="console.log()">
+    <v-row class="ma-0">
+      <v-col cols="7" class="pa-0 border-e-md">
+        <v-card-title class="text-h6 text-center">
+          {{ equipment.name }}
+        </v-card-title>
+        <v-card-text class="d-flex flex-row">
+          <v-progress-circular :model-value="equipment.oee" size="150" width="20" :color="getColorState(equipment.oee, equipment.isPaused)" class="mx-auto text-h6">
```

Comment:
- This line is long enough that I fell the attributes should be on separate lines
- If the parent component if flex, you shouldn't need `mx-auto`
- There should be some label specifying what this quantity is (i.e. OEE)